glass/fields.py

```diff
@@ -293,13 +288,13 @@ def generate_gaussian(
 
 
 def generate_lognormal(
-    gls: Cls,
+    gls: npt.NDArray[np.float64],
     nside: int,
     shift: float = 1.0,
     *,
     ncorr: int | None = None,
     rng: np.random.Generator | None = None,
-) -> collections.abc.Generator[npt.NDArray[typing.Any], None, None]:
+) -> collections.abc.Generator[npt.NDArray[np.complex128]]:
```

Comment:
see above

Reply:
`mypy` actually told me this one before I got around to this comment!